src/uu/ls/Cargo.toml

```
@@ -15,15 +15,17 @@ edition = "2018"
 path = "src/ls.rs"
 
 [dependencies]
+indoc = "1.0.3"
+locale = "0.2.2"
+chrono = "0.4.19"
 clap = "2.33"
 lazy_static = "1.0.1"
 number_prefix = "0.4"
 term_grid = "0.1.5"
 termsize = "0.1.6"
-time = "0.1.40"
 unicode-width = "0.1.5"
-uucore = { version=">=0.0.7", package="uucore", path="../../uucore", features=["entries", "fs"] }
-uucore_procs = { version=">=0.0.5", package="uucore_procs", path="../../uucore_procs" }
+uucore = { version = ">=0.0.7", package = "uucore", path = "../../uucore", features = ["entries", "fs"] }
+uucore_procs = { version = ">=0.0.5", package = "uucore_procs", path = "../../uucore_procs" }
```

please revert this unrelated change. thanks

```suggestion
uucore = { version=">=0.0.7", package="uucore", path="../../uucore", features=["entries", "fs"] }
uucore_procs = { version=">=0.0.5", package="uucore_procs", path="../../uucore_procs" }
```

---

@sylvestre This is rustfmt formatting the toml.